server/events.py

```diff
@@ -102,17 +109,20 @@ class ErrorData(EventData):
     message: str
 
 
-class Event(BaseModel):
-    """A WebSocket event.
+class EventRequest(BaseModel):
+    """A WebSocket request event.
+
+    This represent a request made from the client to the server.
 
     Fields:
         type: The type of the event.
         data: The data of the event.
+        status_code (optional): The status code of the event. Only required when
+            sending an event from the server to the client.
```

Comment:
Don't need in EventRequest

Reply:
Fixed in 87edde6328f2a6ec25169ab20d22a8747c4eccc1.